.github/workflows/ci.yml

```diff
@@ -0,0 +1,110 @@
+name: ci
+on:
+  push:
+    branches:
+      - "**"
+  workflow_dispatch:
+    inputs:
+      deploy-to-dev:
+        description: "Whether to deploy to dev"
+        type: boolean
+        required: true
+        default: false
+
+defaults:
+  run:
+    # NOTE: A bit stricter than the default bash options used by GitHub Actions
+    # (bash --noprofile --norc -e -o pipefail {0})
+    shell: bash --noprofile --norc -euo pipefail {0}
+
+# NOTE: Set concurrency for the current workflow to 1
+concurrency: ci-${{ github.ref }}-${{ github.workflow }}
+
+jobs:
+  build-and-deploy:
+    timeout-minutes: 60
+    runs-on: ubuntu-22.04
+    permissions:
+      actions: read
+      contents: read
+      id-token: write
+    steps:
+      - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+
+      - uses: capralifecycle/actions-lib/check-runtime-dependencies@7eaefbf567a011164b4085366190807b4ddce7bc # v1.2.0
+
+      - uses: capralifecycle/actions-lib/parse-config@7eaefbf567a011164b4085366190807b4ddce7bc # v1.2.0
+        id: config
+        with:
+          config-file: ".ldp.json"
+
+      - uses: actions/setup-node@60edb5dd545a775178f52524783378180af0d1f8 # v4.0.2
+        with:
+          node-version: 20.9.0
+
+      - name: install dependencies
+        run: npm ci
```

Comment:
Hva med `--legacy-peer-deps` fra Jenkins-flyten, er det nødvendig her?

Reply:
Burde ha notert noe rundt denne ja. Jeg droppet det siden det virket som etterslep vi kunne og burde fjerne. Det var ikke dokumentert hvorfor flagget ble brukt i [commiten det ble introdusert](https://github.com/capralifecycle/webapp-baseline/commit/7a68876c7acda28a403d526f99d662bcfd076e4f), og [dokumentasjonen](https://docs.npmjs.com/cli/v10/using-npm/config#legacy-peer-deps) beskriver det som ikke anbefalt og en workaround for å ignorere konflikter i peerDependencies under `npm install`.

Pakkene installerte fint lokalt uten overstyringen, og ser ut som det gikk fint i CI og, så tolket det som at hvilke konflikter som enn eksisterte i dependencies i 2022 ikke lenger er der